CMakeLists.txt

```diff
@@ -1,36 +1,22 @@
-# We do not really use C++, but CXX is still required for using the
-# raspberrypi/pico-sdk. This also enable C++ support.
-enable_language(C CXX)
+cmake_minimum_required(VERSION 3.13)
 
-set(PICO_ICE_SDK_PATH ${CMAKE_CURRENT_LIST_DIR})
-list(APPEND PICO_BOARD_HEADER_DIRS ${PICO_ICE_SDK_PATH}/include/boards)
+if (NOT TARGET pico_ice_sdk)
+    if (NOT DEFINED PICO_SDK_PATH)
+        set(PICO_SDK_PATH "${CMAKE_CURRENT_LIST_DIR}/pico-sdk")
+    endif()
```

Comment:
Since we are having it as a submodule, it makes sense to provide a default value to it, flexible and easy.